README.md

```diff
@@ -98,6 +99,8 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 - [httpexpect](https://github.com/gavv/httpexpect/labels/help%20wanted) _(label: help wanted)_ <br> End-to-end HTTP and REST API testing for Go.
 - [Killgrave](https://github.com/friendsofgo/killgrave/labels/good%20first%20issue) _(label: good first issue)_ <br> Simple way to generate mock servers in Go.
 - [lxd](https://github.com/lxc/lxd/labels/easy) _(label: easy)_ <br> System container and virtual machine manager.
+- [Prometheus](https://github.com/prometheus/prometheus/issues?q=is%3Aissue+is%3Aopen+label%3A%22low+hanging+fruit%22) _(label: low hanging fruit)_ <br> Prometheus, a Cloud Native Computing Foundation project, is a systems and service monitoring system. It collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts when specified conditions are observed.
```

Comment:
I see this repo also has "hacktoberfest" label and they do not overlap with "low-hanging-fruit" much. Can we check with any of the maintainers on what is the strategy (if any) they have for new contributors?